README.md

```diff
@@ -1,2 +1,49 @@
-# POAP-APP
-The web app for Proof of Attendance Protocol 
+# POAP APP
+
+The web app for Proof of Attendance Protocol
+
+## Requirements
+- Node.js `v18.16.0+`
+- Yarn `v1.22.19+`
+- Running POAP API server (backend), for details see [here](https://github.com/rikublock/POAP-API)
+
+## Getting Started
+- install dependencies with `yarn`
+- rename `.env.example` to `.env` (change values as needed)
+- run the backend
+- run the app with `yarn start`
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `yarn`
+
+Install dependencies.
+
+### `yarn start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.\
+You will also see any lint errors in the console.
+
+### `yarn test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `yarn build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+## Supported Wallets
+- Xumm (installation details [here](https://xumm.app/))
```

Comment:
nit: Xumm is rebranding as `Xaman` so maybe it'd be good to rename them in this project? (With a note like "Xaman, formerly named Xumm, ..."